ring_doorbell/__init__.py

```diff
@@ -29,7 +31,9 @@
 class Ring(object):
     """A Python Abstraction object to Ring Door Bell."""
 
-    def __init__(self, username, password, debug=False, persist_token=False,
+    def __init__(self, username, password,
```

Comment:
I would suggest we change the Ring object to just pass in the auth as an object.